igcollect/ping.py

```diff
@@ -4,83 +4,90 @@
 Copyright (c) 2020 InnoGames GmbH
 """
 
-import _thread
+# this script is used to collect ping data to evaluate availability and latency
+# of our network (this should replace smokeping in the longrun)
+#
+# Please Not that the package fping has to be installed on your Linux server!
```

Comment:
How safe is it to rely on `fping` and how important is it? We are already using multiprocessing, how balanced are the pros/cons using `fping` over `ping` here?

Reply:
fping is much faster than multi threading ping itselve we allready tested it ^^